vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -75,8 +75,9 @@
         %(AdditionalIncludeDirectories);
       </AdditionalIncludeDirectories>
       <PreprocessorDefinitions>_WIN32;_CRT_SECURE_NO_WARNINGS;FOLLY_NO_CONFIG;NOMINMAX;RN_EXPORT=;JSI_EXPORT=;WIN32;_WINDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(EnableHermesInspector)' == 'true'">HERMES_INSPECTOR_FOLLY_KLUDGE;HERMES_ENABLE_DEBUGGER;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ForcedUsingFiles />
-      <DisableSpecificWarnings>4715;4146;4251;4800;4804;4305;4722;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <DisableSpecificWarnings>4244;4018;4541;4267;4715;4146;4251;4800;4804;4305;4722;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
most of these new ones will break SDL

Reply:
These are coming from hermes debugger headers unfortunately .. We have a task in hand to take care of compiler warnings in Hermes.. Will clear these alongwith